pkgs/desktops/gnome/extensions/argos/default.nix

```diff
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "argos-unstable";
-  version = "20230404";
+  version = "20230926";
```

Comment:
I don't think this is following the format described in the contributing guide

Reply:
It was following an older format (before #149667). Adjusted.